drivers/syslog/vsyslog.c

```diff
@@ -50,6 +50,16 @@
 #include <nuttx/streams.h>
 #include <nuttx/syslog/syslog.h>
 
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+static const char * priority_str[] =
+  {
```

Comment:
Shouldn't this be enclosed in

    #if defined(CONFIG_SYSLOG_PRIORITY)

Reply:
Yes, it is.

Seems like I pushed the wrong commit. There are also formatting fixes.

Pushing again now.

Reply:
@patacongo It seems it is correct now. Sorry for this!

Reply:
need add FAR and g_ too:
```
static FAR const char * g_priority_str[] =
```

Reply:
Done